### PR TITLE
Adding manual test template

### PR DESCRIPTION
**Pull request by ApocalypticFish**
This adds a template for testing new builds which can be added as a part of the release process.

## What's this?
This adds a template in github for testing issues

### Relationships
None

Closes #
Nothing

## Why do we want this?
Because this means that there is a paper trail for testing in github that can be linked to release projects

## How is it done?
It adds a document in the templates folder which allows a testing checklist to be created for each issue.

### Why not another way?
Because this means it can be tracked

### CODE REVIEW

**Comment by heytherewill** [comment]
If this is not a WIP, please update the Pull request template to let us know what needs to be reviewed

**Comment by ApocalypticFish** [comment]
Thanks, @heytherewill I updated that now.

File: .github/ISSUE_TEMPLATE/Manual UI Test Template [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+[] Archiving
+[] Create new project on web whilst mobile offline
+[] Tasks
+[] Try creating new entries on web whilst mobile offline
+[] Edit client on web whilst mobile offline
+[] Try stopping timer on web while mobile offline
+[] Delete project on web whilst mobile offline
+[] Make changes to projects on web
+
+*Testing in different timezones*
```
Comment (heytherewill) [inline .github/ISSUE_TEMPLATE/Manual UI Test Template]: Since these headers have no items in them, shouldn't the headers themselves have checkboxes?

Comment (ApocalypticFish) [reply .github/ISSUE_TEMPLATE/Manual UI Test Template]: Yeah, good idea

**Comment by ApocalypticFish** [comment]
@amulware I added the other parts but have no idea where to do this or what you're referring to? by "first headline (which currently does not exist), similar to the other templates." 

I worked backwards from the existing templates

**Comment by amulware** [comment]
The other templates have a generic first headline describing what kind of issue this is and including the template emoji, which makes it easy to see what you're looking at at a glance.

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+---
+name: ":penguin: Manual Test Template"
+about: A manual testing checklist that can be assigned to a release to track the progress of testing.
+
+---
+
+
+## []Testing sign in area
```
Comment (amulware) [inline .github/ISSUE_TEMPLATE/d-manual-test-template.md]: I don't know if checkboxes in headlines work, but in either case, you'd most likely need spaces around it.

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+[] Test privacy policy link
+[] Test signin with Google signin
+[] Test signup with Google signin
+[] Test forgotten password link
+[] Test forgotten password link with invalid email
+[] Is country selection automatically selecting the right country?
+[] Is country selection allowing manual selection
+[] Test email field with invalid email
+[] Test email field with mixed character sets (including various languages)
+[] Test password field for old/changed password
```
Comment (amulware) [inline .github/ISSUE_TEMPLATE/d-manual-test-template.md]: These render as a single paragraph, you need to add dashes to turn them into bullet points. And potentially spaces between the brackets to turn them into check boxes.

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+[] Switch workspaces
+[] Go offline then switch workspaces (Airplane mode)
+[] Switch between workspaces while timer running
+[] Is data displaying correctly in web?
+[] Force sync then switch
+[] Leave a workspace
+[] Be removed from a workspace while offline
+[] Be removed from a workspace with an active entry running
+
+##*[]Testing Interactions/Sync between this app and others*
```
Comment (amulware) [inline .github/ISSUE_TEMPLATE/d-manual-test-template.md]: There are still italics here which break the headlines.

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md (hunk 2 above)
Comment (ApocalypticFish) [reply .github/ISSUE_TEMPLATE/d-manual-test-template.md]: They do not. Not in any way I could find. So I scrapped them.

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md (hunk 4 above)
Comment (ApocalypticFish) [reply .github/ISSUE_TEMPLATE/d-manual-test-template.md]: Yes, that is not good.

**Comment by ApocalypticFish** [comment]
The preview looks good to me now.

**Pull request merged**